extensions/Gemini/python/src/aiconfig_extension_gemini/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .Gemini import GeminiModelParser
+
+__ALL__ = ["GeminiModelParser"]
```

Comment:
I think this is supposed to be lowercase
```suggestion
__all__ = ["GeminiModelParser"]
```